ExomeDepth/make_BEDdetail.py

```diff
@@ -390,6 +403,13 @@ def make_bed_detail(args, event_dic, children, parents):
         '--correlqc', default=settings.correlation, type=float,
         help='Threshold for minimum allowed correlation score (default = 0.98)'
     )
+    parser.add_argument(
+        '--random_off', action='store_true',
+        help=(
+            'Do not apply randomisation in VCF-file order.'
+            ' This is usefull for validation purposes if same results are needed (default = random selection on'
```

Comment:
Missing a ")" at the end of the sentence

Reply:
I even detected 2 of those :)